spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -1295,6 +1295,31 @@ The configuration passed to the `IncomingConnectorFactory` and `OutgoingConnecto
 * `channel-name` attribute indicating the name of the channel being configured,
 * `connector` attribute must match the name given to the `@Connector` qualifier.
 
+==== Connector attribute
+
+To help tools (IDEs, documentation generator) to extract the configuration of each connector, the specification recommends the usage of the `org.eclipse.microprofile.reactive.messaging.spi.ConnectorAttribute` annotation.
```

Comment:
I am wondering whether we could add an optional TCK. What do you think?

Reply:
I don't believe we can, what would it be? 
The implementations do not do anything with these attributes.

Reply:
okay, I have not thought of a solution as yet, but I thought you might know. If you don't believe we could, it is okay.

Reply:
To help tools (IDEs, documentation generator) to extract the configuration of each connector, the specification recommends the usage of the `org.eclipse.microprofile.reactive.messaging.spi.ConnectorAttribute` annotation. =>

To help tools (IDEs, documentation generator) to extract the configuration of each connector, the specification provides the `org.eclipse.microprofile.reactive.messaging.spi.ConnectorAttribute` annotation for implementations to create a good ecosystem with the tools. However, the support for this SPI is not mandatory.

Reply:
Fixed.